src/realm/CMakeLists.txt

```diff
@@ -0,0 +1,324 @@
+set(REALM_SOURCES
+    alloc.cpp
```

Comment:
We could, if the folder didn't include also other *.cpp files such as the importer, realmd and so on. Changing the directory structure is also on the roadmap, but not for this PR.